events_task/lib/my_app_web/templates/event/show.html.eex

```diff
@@ -0,0 +1,31 @@
+<h1>Show Event</h1>
+
+<ul>
+
+  <li>
+    <strong>Description:</strong>
+    <%= @event.description %>
+  </li>
+
+  <li>
+    <strong>Timestamp:</strong>
+    <%= @event.timestamp %>
+  </li>
+
+</ul>
+<span><%= link "Back", to: "/" %></span>
+<%= if @current_user do %>
+	<%= if @current_user.id == @event.author_id do %>
```

Comment:
Same as above - you are putting security checks only into templates. They should be here not to show unnecessary elements that uses can't click on, but they should be duplicated in controllers - right now any non-author user, even if he doesn't see 'Edit' button, can add `/edit` to url and successfully edit event that does not belong to him.